workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-checkout-step/client-extension.yaml

```diff
@@ -0,0 +1,26 @@
+assemble:
+    - fromTask: bootJar
+    - from: build/resources/main/assets
+      into: static
+liferay-sample-custom-checkout-step:
+    actionURL: /liferay-sample-custom-checkout-step
```

Comment:
shouldn't this be something easier, like `/process-action` or even `/process/action`

Reply:
this is actually up to who develop the custom step, I just wanted to use the a URL bounded to the context